### ISSUE TITLE
Fan control for RAM

### ISSUE THREAD

**Issue by 3ighty2O** [Rem0o/FanControl.Releases#1981]
I wanna adjust fan according to my ram's temperature but I can't find one that allows ram (dimm) temp as input. I have mine W.C and would love to keep ram's temp below 35*C. At 80%, the fan do just that, but the noise is noticeable.

**Comment by Rem0o** [comment]
Not all ram DIMM offer temperature probes. If yours are compatible, make sure you got the Memory option checked in the sensor settings dialog.

Otherwise, see https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility

**Comment by 3ighty2O** [comment]
I'm using G.skill which has temp probe. Is PECI 1 Calibrated is the name of the Ram sensor?

**Comment by JanB97** [comment]
I also have Gskill memory, it's not showing up.
With HWInfo i can see the temp of the RAM.
Memory is selected as sensor.

PCIE ist the temp sensor on PCIE Slot 1 (usually gpu) and from the mainboard

**Comment by Ahli** [comment]
LibreHardwareMonitor merged a PR to read RAM temperatures today: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/1743

**Comment by Rem0o** [comment]
Will be in the next update. In the mean time you can take the dll directly from lhm it should be plug and play.

**Comment by spec-chum** [comment]
> Will be in the next update. In the mean time you can take the dll directly from lhm it should be plug and play.

Not quite, I got it working but I had to do a little bit of editing with the LHM project file.

You have to compile the LHM dll to `net48`, as it doesn't by default, and also add `RAMSPDToolkit-NDD.dll` to the FanControl folder - seems to work fine, tho.
